packages/runtime/container-runtime/src/test/opLifecycle/outbox.spec.ts

```diff
@@ -494,13 +498,16 @@ describe("Outbox", () => {
 		);
 	});
 
-	it("Compress only if compression is enabled", () => {
+	it("Compress only if compression and grouping are enabled", () => {
```

Comment:
Not sure why "only" is in the title, this test doesn't cover the other case(s).
```suggestion
	it("Compress if compression and grouping are enabled", () => {
```

Reply:
Applied changes